Formula/fatsort.rb

```diff
@@ -1,9 +1,9 @@
 class Fatsort < Formula
   desc "Sorts FAT16 and FAT32 partitions"
   homepage "https://fatsort.sourceforge.io/"
-  url "https://downloads.sourceforge.net/project/fatsort/fatsort-1.6.2.605.tar.xz"
+  url "https://github.com/sh-khashimov/fatsort/archive/master.tar.gz"
```

Comment:
that's not a correct version

Reply:
you are absolutely right, I have emailed fatsort original author and asked him if he can incorporate the changes modified in that github repo.

Reply:
I have already marked the PR as do not merge and will wait for him to comment in here (will close this next weekend.)